### ISSUE TITLE
Different Header between iOS and Android

### ISSUE THREAD

**Issue by lorenzoPrimi** [ooni/probe#1194]
The test result summary header is different between iOS and Android, on this last one it doesn't have the "Websites" and "Apps" labels.
We need to change one of the platform to unify them. According to @hellais it's better to remove that word as it doesn't add any value.

@sarathms you might want to follow this to adapt the Desktop app as well

Android:
![Screenshot_1591716201.png](https://images.zenhubusercontent.com/589f552b51aa2c3057742305/7798ec74-1015-4898-9126-ed772416f4bb)
![Screenshot_1591716190.png](https://images.zenhubusercontent.com/589f552b51aa2c3057742305/a61046ac-190f-4c79-aab8-ca34a143faa6)

iOS:
![Simulator Screen Shot - iPhone 11 Pro Max - 2020-06-09 at 17.13.26.png](https://images.zenhubusercontent.com/589f552b51aa2c3057742305/77b63c49-b4a4-4d9b-862a-91dd1ffe962c)
![Simulator Screen Shot - iPhone 11 Pro Max - 2020-06-09 at 17.13.22.png](https://images.zenhubusercontent.com/589f552b51aa2c3057742305/50222461-a256-42d9-9084-7e912392d7a3)

**Comment by lorenzoPrimi** [comment]
The desktop app has the top and bottom strings, so I guess I'll have to edit Android, not iOS


![Screenshot 2020-07-05 at 16.31.26.png](https://images.zenhubusercontent.com/589f552b51aa2c3057742305/aa67776b-3aa6-41e9-92d2-e73c760aab71)
![Screenshot 2020-07-05 at 16.31.35.png](https://images.zenhubusercontent.com/589f552b51aa2c3057742305/474fbe38-5135-453a-a7cc-a0cc539d4beb)
![Screenshot 2020-07-05 at 16.32.02.png](https://images.zenhubusercontent.com/589f552b51aa2c3057742305/11020a60-f59c-4e68-9b0d-cf4e851fff32)
![Screenshot 2020-07-05 at 16.32.14.png](https://images.zenhubusercontent.com/589f552b51aa2c3057742305/4c4e4108-09e5-495b-a368-e951f45ebcac)

**Comment by hellais** [comment]
I think we can very well get rid of the bottom label on the first 3 screens you posted and still preserve the meaning.

On mobile this is displayed contextually to the title saying the name of the test, so it's actually a repetition.

That is to say, that in the following screen it reads:

![Screenshot 2020-07-06 at 10 55 05](https://user-images.githubusercontent.com/424620/86575272-481b8d00-bf77-11ea-8170-4aa686500029.png)

Websites: 42 Tested, 2 blocked, 38 accessible. While if you had the websites label it feels much more verbose and noisy, while not adding any additional value.

I suggest keeping it in the performance test header, since there the unit changes depending on the box and we need to clarify it.

So to recap, drop the bottom label from: Circumvention, Websites & IM test groups, but keep it in performance.

**Comment by lorenzoPrimi** [comment]
I need to coordinate with @sarathms as we need to drop these strings from the copy and it can create problem if we don't do this at the same time.

**Comment by hellais** [comment]
> as we need to drop these strings from the copy 

Why do we need to drop them from the copy? Can you not just stop using one of the labels in the mobile app, but we keep it for the time being in the desktop app?

I think in the desktop app it has still some sense to exist, because on desktop the header has the date instead of having the name of the test group.

We should eventually align them, but I would not want this to be a blocker on moving forward an improvement on mobile.

**Comment by lorenzoPrimi** [comment]
Ok, so for now I will only mark these strings as "unused" in the copy like we did for the middle boxes strings without removing them.
We might want to open an issue for desktop to don't forget to unify the design
